assets/css/style.css

```diff
@@ -1,2 +1,183 @@
 
+body {
+    font-family:'Raleway', sans-serif;
+}
+body a {
+    text-decoration: none;
+    color: black;
+    font-weight: 500;
+}
+body a:hover {
+    color: #0065FC;
+}
 
+header {
+    flex-direction: column;
+    font-size: 2.5rem;
+    padding-bottom: 2rem;
+}
+header img {
+    width: 25%;
+    height: auto;
+}
+header ul {
+    width: 100%;
+    justify-content: center;
+}
+header li {
+    padding: 3rem 9rem;
+    border-bottom: solid 6px #F2F2F2;
+}
+header li:hover {
+    border-bottom: solid 6px #0065FC;
+}
+
+#inscription {
+    position: absolute;
+    top: 90px;
+    right: 50px;
+    color: #0065FC;
+}
+h1 {
+    font-size: 3.4rem;
+}
+h2 {
+    font-size: 3.5rem;
+    font-weight: bold;
+    padding: 4rem 0;
+}
+h3  {
+    font-size: 2.6rem;
+    font-weight: 700;
+}
+#section-Search p {
+    padding: 0 3rem 3rem 3rem;
+    font-size: 1.2rem;
+}
+main i {
+    font-size: 3rem;
+    padding: 2.5rem;
+    background-color: #DEEBFF;
+}
+form {
+    border: solid 3px #F2F2F2;
+    border-radius: 30px;
+    margin: 3rem;
+}
+input[type=text]{
+    font-size: 3rem;
+    margin-left: 1rem;
+    border: none;
+}
+.fa-map-marker-alt  {
+    background-color: #f2f2f2;
+    border-top-left-radius: 30px;
+    border-bottom-left-radius: 30px;
+}
+.fa-search {
+    border-radius: 30px;
+    background-color: #0065FC;
+    color: white;
+}
+#section-Filter .fas {
+    border-radius: 50%;
+    margin-right: 8px;
+    color: #0065FC;
+}
+#section-Filter li {
+    font-size: 3rem;
+    border: solid 6px lightgrey;
+    border-radius: 70px;
+    padding-right: 1.9rem ;
+    align-items: baseline;
+    margin: 10px;
+}
+#section-Filter li:hover {
+    background-color: #DEEBFF;
+}
+.fa-info {
+    float: left;
+    border: solid 0.1px lightgrey;
+    background-color: white;
+    font-size: 10px;
+    padding: 1rem 1.5rem 1rem 1.5rem;
+    position: relative;
+    right: 1rem;
+}
+#section-lesPlusPopulaires {
+    order: 1;
+    background-color: #F2F2F2;
+    margin: 20px 0 20px 0;
+    width: 100%;
+}
+#section-hebergements {
+    order: 2;
+}
+#section-activites {
+    order: 3;
+}
+.fa-star {
+    font-size: 25px;
+    padding: 0;
+}
+.fa-chart-line {
+    float: right;
+    padding: 0;
+    background-color: #F2F2F2;
+}
+#section-lesPlusPopulaires img {
+    height: 24rem;
+    width: 22rem;
+    object-fit: cover;
+    border-top-left-radius: 30px;
+    border-bottom-left-radius: 30px;
+
+}
+#section-lesPlusPopulaires a {
+    display: flex;
+}
+ article {
+    background-color: white;
+    border-radius: 30px;
+    border: solid 12px white;
+    margin: 2rem 0;
+}
+.container-infos-articles-lesPlusPopulaires {
+    padding: 2rem;
+    align-content: space-between;
+    
+}
+#section-lesPlusPopulaires p {
+    font-size: 2.3rem;
+}
+.fa-star {
```

Comment:
don't use selector with icon class, what if you change icon later ? And you already have a block for this, no duplicate